1.Core/UIElements/Actions/Actions.cs

```diff
@@ -21,5 +21,6 @@ public static void DragAndDrop(IWebElement source, IWebElement target)
     public static void NavigateTo(string url)
     {
         GenericWebDriver.Instance.Navigate().GoToUrl(url);
+        GenericWebDriver.Instance.Navigate().GoToUrl("https://todo.ly/");
```

Comment:
This line seems to be unneeded. You should set the URL from an environment variable.

Reply:
hi sebastian :) 
Actually it's needed because the main url here is the following: https://hub.browserstack.com/wd/hub/
in order to make the connection with browserstack. So we need to pass the next url which is https://todo.ly/

This is more or less how code needs to look like:
IWebDriver _**driver**_ = new RemoteWebDriver(new Uri("_https://hub.browserstack.com/wd/hub/_"), capability);
_**driver**_.Navigate().GoToUrl("_https://www.duckduckgo.com_");

Reply:
now it's fixed :)